azext_edge/edge/providers/edge_api/certmanager.py

```diff
@@ -0,0 +1,10 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from .base import EdgeResourceApi
+
+
+CERTMANAGER_API_V1 = EdgeResourceApi(group="cert-manager.io", version="v1", moniker="certmanager")
```

Comment:
do you need to put this in help?

Reply:
nice catch~ added 
<img width="429" alt="image" src="https://github.com/user-attachments/assets/4cbeb9b2-adf2-442c-b70f-c97a4aa7219f" />